client/styles/device.ts

```diff
@@ -0,0 +1,22 @@
+const size = Object.freeze({
```

Comment:
if this is specifically used for screen widths and not heights, maybe `width` is a better name?

Reply:
ya that makes sense :)

Reply:
actually, I'm looking at it again and I feel like `size` makes more sense as we could in the future export heights if need be from the same set of constants

Reply:
Would these sizes be reusable for heights though? Or should we have two objects, `width` and `height`?

Reply:
hmm ya true